### PR TITLE
Fixed Reply issue

### PR DESCRIPTION
**Pull request by TyagiLikeMinds**

### CODE REVIEW

File: lib/src/utils/chatroom/conversation_utils.dart [hunk 1]
```diff
@@ -28,6 +28,7 @@ List<Conversation>? addTimeStampInConversationList(
           isTimeStamp: true,
           answer: key,
           communityId: communityId,
+          chatroomId: 0,
```
Comment (divyanshgandhilm) [inline lib/src/utils/chatroom/conversation_utils.dart:31]: Why are we adding 0 as chatroomId?

File: lib/likeminds_chat_mm_fl.dart [hunk 2]
```diff
@@ -86,7 +87,7 @@ class LMChat extends StatelessWidget {
   }
 
   firebase() async {
-    bool? isDebug = const bool.fromEnvironment('DEBUG');
+    // bool? isDebug = const bool.fromEnvironment('DEBUG');
```
Comment (divyanshgandhilm) [inline lib/likeminds_chat_mm_fl.dart:90]: Were we not using this bool?

File: pubspec.yaml [hunk 3]
```diff
@@ -14,8 +14,8 @@ dependencies:
     sdk: flutter
 
   ## LikeMinds Chat SDK
-  likeminds_chat_fl: 1.1.0
-     #path: ../LikeMinds-Flutter-GroupChat-SDK
+  likeminds_chat_fl: 1.1.1
```
Comment (divyanshgandhilm) [inline pubspec.yaml:17]: Don't build like this while testing, otherwise fine for now

**Review by divyanshgandhilm — CHANGES_REQUESTED** [review]

File: lib/src/utils/chatroom/conversation_utils.dart (hunk 1 above)
Comment (TyagiLikeMinds) [reply lib/src/utils/chatroom/conversation_utils.dart:31]: Dummy conversation model for timestamp

File: lib/likeminds_chat_mm_fl.dart (hunk 2 above)
Comment (TyagiLikeMinds) [reply lib/likeminds_chat_mm_fl.dart:90]: Not needed here

File: pubspec.yaml (hunk 3 above)
Comment (TyagiLikeMinds) [reply pubspec.yaml:17]: okayy

**Review by divyanshgandhilm — APPROVED** [review]

**Pull request merged**